pdr_backend/cli/cli_arguments.py

```diff
@@ -39,14 +43,21 @@
 
 HELP_DOT = """
 To pass args down to ppss, use dot notation.
-Example: pdr lake ppss.yaml sapphire-mainnet --lake_ss.st_timestr=2023-01-01 --lake_ss.fin_timestr=2023-12-31
+Example: pdr lake raw|etl update ppss.yaml sapphire-mainnet
+pdr lake raw|etl drop ppss.yaml sapphire-mainnet 2023-06-01
+pdr lake describe --HTML ppss.yaml sapphire-mainnet
+pdr lake validate ppss.yaml sapphire-mainnet
 """
 
 HELP_OTHER_TOOLS = """
 Power tools:
   pdr multisim PPSS_FILE
+  pdr arima_plots PPSS_FILE [--debug_mode False]
   pdr deployer (for >1 predictoor bots)
-  pdr lake PPSS_FILE NETWORK
+  pdr lake raw|etl update PPSS_FILE NETWORK
+  pdr lake raw|etl drop PPSS_FILE NETWORK ST
```

Comment:
lake commands are duplicate, they are both mentioned in `HELP_OTHER_TOOLS` and `Inspect and manage lake`

Reply:
I don't see a problem with this. I don't see why it should be exclusive... but if it is, then i'll move more stuff from "power tools" to "lake"